tests/test_db.py

```diff
@@ -1,64 +1,92 @@
-"""
-test_db
--------
+"""Tests of alpenhorn.db module."""
 
-Tests for `alpenhorn.db` module.
-"""
-
-import peewee as pw
 import pytest
+import peewee as pw
+import threading
+
+from alpenhorn import db
+
+
+def test_db(dbproxy):
+    """Test starting the default DB."""
+    assert not db.threadsafe
+    assert issubclass(dbproxy.obj.__class__, pw.Database)
+
+
+def test_chimedb(use_chimedb, dbproxy):
+    """Test starting the default DB."""
+    assert db.threadsafe
+    assert issubclass(dbproxy.obj.__class__, pw.Database)
+
+
+def test_chimedb_concurrency(use_chimedb, dbproxy):
+    """Test concurrency in the chimedb db module."""
+
+    # Test table
+    class Values(db.base_model):
+        value = pw.IntegerField()
+
+    # Create and populate
+    dbproxy.create_tables([Values])
+    assert dbproxy.get_tables() == ["values"]
+    Values.insert_many([{"value": 12}, {"value": 34}, {"value": 56}]).execute()
+
```

Comment:
For understanding the exact tests I think it might be useful to test the db contents at this point.
```suggestion

     assert list(Values.select().dicts()) == [
         {"id": 1, "value": 12},
         {"id": 2, "value": 34},
         {"id": 3, "value": 56},
     ]
     
```